api/rpc/account/account.proto

```diff
@@ -146,7 +146,8 @@ message SignUpRequest {
 
 message VerificationRequest {
   string name = 1;
-  string code = 2;
+  string password = 2;
```

Comment:
Since we should only be transmitting a hash of the password, we should specify an appropriately sized fixed length byte array (probably 12 bytes, but you and @generalhenry should confirm and coordinate on this).